src/main/java/io/github/railroad/platform/forge/ForgeVersion.java

```diff
@@ -15,10 +15,10 @@
 
 /**
  * @author ChAoS
- *
- * Class ForgeClass can be used as a list of forge version fetching result,
- * you can only get it by using method {@code downloadVersion()}.
- *
+ * <p>
+ * Class ForgeClass can be used as a list of forge version fetching result, you can only get it by using method {@code
+ * downloadVersion()}.
+ * <p>
  * TODO: Sorted version list still unavailable until {@code SemanticVersion.parse(String value)} is fully handled.
  */
 public class ForgeVersion {
```

Comment:
You can switch this to use Java NIO and remove all the weird XML stuff. Ill do it in my PR.

Reply:
Yep.